swagger_to_sdk_config.schema.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "type": "object",
+    "properties": {
+        "$schema": {
+            "enum": [
+                "https://raw.githubusercontent.com/lmazuel/swagger-to-sdk/master/swagger_to_sdk_config.schema.json"
+            ]
+        },
+        "meta": {
+            "type": "object",
+            "properties": {
+                "version": {
+                    "enum": [
+                        "0.2.0"
+                    ],
+                    "description": "The version must be 0.2.0"
+                },
+                "after_scripts": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "minLength": 1
+                    },
+                    "description": "List of commands to execute after the generation is done. Will be executed in the order of the list. Current working directory will be the cloned path. See also \"envs\" node."
+                },
+                "autorest_options": {
+                    "$ref": "#/definitions/autorest_options"
+                },
+                "envs": {
+                    "type": "object",
+                    "patternProperties": {
+                        "sdkrel:.*": {
+                            "type": "string",
+                            "description": "All options prefixed by \"sdkrel:\" can be a relative path that will be solved against SDK folder before being sent to Autorest."
+                        }
+                    },
+                    "additionalProperties": {
+                        "oneOf": [
+                            {
+                                "type": "string"
+                            },
+                            {
+                                "type": "boolean"
+                            },
+                            {
+                                "type": "number"
+                            }
+                        ]
+                    },
+                    "description": "Environment variables for after_scripts. All options prefixed by \"sdkrel:\" can be a relative path that will be solved against SDK folder before being sent to the scripts."
+                },
+                "advanced_options": {
+                    "type": "object",
+                    "properties": {
+                        "clone_dir": {
+                            "type": "string",
+                            "minLength": 1,
+                            "description": "Add more layers of folders to clone the repo, if necessary. Right now, useful for Go only. \"sdkrel:\" will consider this as the final folder path."
+                        }
+                    },
+                    "patternProperties": {
+                        "sdkrel:.*": {
```

Comment:
Does it mean it's required? Because being prefixed by "sdkrel:" is optional

Reply:
No. That means that if the property's name matches the provided pattern, then the property's value must match the provided schema. It doesn't mean that it's required.